py-polars/polars/internals/series.py

```diff
@@ -307,16 +297,20 @@ def _comp(self, other: Any, op: str) -> "Series":
         if isinstance(other, datetime) and self.dtype == Datetime:
             ts = _datetime_to_pl_timestamp(other)
             f = get_ffi_func(op + "_<>", Int64, self._s)
-            return wrap_s(f(ts))  # type: ignore
```

Comment:
Strictly speaking not related to this PR, but the type: ignore is flagging us that we need to deal with `f is None`.

Reply:
Yes, except, I am certain that function exists. So the check is redundant IMO.

Reply:
I agree that ignoring the typing error is cleaner than adding an additional runtime check, reverted.